cocos/input/types/event/event-mouse.ts

```diff
@@ -100,6 +100,17 @@ export class EventMouse extends Event {
      */
     public movementY = 0;
 
+    /**
+     * @en Set whether the event is triggered only by the top node, which is true by default.
+     * If set to false, the event is allowed to be dispatched to nodes at the bottom layer.
+     * NOTE: Setting to false will reduce the efficiency of event dispatching.
+     *
+     * @zh 设置事件是否只被顶部的节点触发, 默认为 true 。
+     * 如果设置为 false，则事件允许派发给渲染在下一层级的节点。
+     * 注意：设置为 false 会降低事件派发的效率。
+     */
+    public swallowedByTopNode = true;
```

Comment:
swallowEvent is simpler, don't mix node notion in event system

Reply:
I think it's strange to name it this way

you mean event swallow event ?

Reply:
`preventSwallow` could be more meaningful